app/views/layouts/_navbar.html.erb

```diff
@@ -19,19 +19,19 @@
       </ul>
       <ul class="nav navbar-nav navbar-right">
         <li>
-          <% if current_user %>
-            <%= link_to "Log out", logout_path, method: :delete%>
-          <% else %>
-            <li class="active"><a href="#" data-toggle="modal" data-target="#loginModal"> Login <span class="sr-only">(current)</span></a></li>
-          <% end %>
-        </li>
-        <li class="dropdown">
-          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false"> Sign Up <span class="caret"></span></a>
-          <ul class="dropdown-menu" role="menu">
-            <li role="active"><a href="#" data-toggle="modal" data-target="#lenderSignUpModal"> As Lender <span class="sr-only">(current)</span></a></li>
-            <li role="active"><a href="#" data-toggle="modal" data-target="#borrowerSignUpModal"> As Borrower<span class="sr-only">(current)</span></a></li>
-          </ul>
-        </li>
+        <% if current_user %>
+          <%= link_to "Log out", logout_path, method: :delete%>
+        <% else %>
+          <li class="active"><a href="#" data-toggle="modal" data-target="#loginModal"> Login <span class="sr-only">(current)</span></a></li>
+        <% end %>
+       </li>
+
+        <% unless current_user %>
+
+          <li role="active"><a href="#" data-toggle="modal" data-target="#lenderSignUpModal"> Sign Up As Lender <span class="sr-only">(current)</span></a></li>
+          <li role="active"><a href="#" data-toggle="modal" data-target="#borrowerSignUpModal"> Sign Up As Borrower<span class="sr-only">(current)</span></a></li>
+
+      <% end %>
```

Comment:
why doesn't this work? We will need to come back to this because a modal would be a preferred signup method